src/Tizen.NUI/src/internal/Interop/Interop.Libraries.cs

```diff
@@ -0,0 +1,27 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.NUI
+{
+    internal static partial class Interop
+    {
+        internal static partial class Libraries
+        {
+            internal const string FrameBroker = "libframe-broker.so.1";
```

Comment:
is it ok to do csharp binding for the native module which is not NUI part?
could you explain what this is for?

Reply:
It is a framework to support app transition animation. It is easy to use in NUI because it supports the function that can be used directly in C#. add to NUI immediately without adding to native code (dali).**
frame_broker : https://github.sec.samsung.net/h-jhun/launcher-service/blob/master/guide/frame-broker.md
frame_provider : https://github.sec.samsung.net/h-jhun/launcher-service/blob/master/guide/frame-provider.md

I think there is no problem this code, but do you think there is any problem with the code?

Reply:
Yes, there will be no problem. I was just curious about whether c# frame broker is provided from c# Application.
thank you for answering.